client_core/src/com/mygdx/game/client_core/ecs/system/CoordinateToPositionSystem.java

```diff
@@ -41,6 +43,8 @@ protected void process(int entityId) {
       toSet = PositionUtil.generateWorldPositionForCoords(coordinatesMapper.get(entityId));
     } else if (subFieldMapper.has(entityId)) {
       toSet = PositionUtil.generateSubWorldPositionForCoords(coordinatesMapper.get(entityId));
+    } else if (technologyMapper.has(entityId)) {
+      toSet = PositionUtil.generateTechnologyPositionForCoords(coordinatesMapper.get(entityId));
```

Comment:
To trzeba jakoś uogólnić już, jak ma to być na podstawie coordów. (choć na razie nie rozumiem czemu)